README.md

```diff
@@ -490,20 +492,54 @@ It is possible to use this library only to load modules (i.e. not to lazy load o
 <script src="https://js.arcgis.com/4.14/" data-esri-loader="loaded"></script>
 ```
 
-### Using the esriLoader Global
+### Without a module bundler
 
-Typically you would [install the esri-loader package](#install) and then `import` the functions you need as shown in all the above examples. However, esri-loader is also distributed as an ES5 [UMD](http://jargon.js.org/_glossary/UMD.md) bundle on [UNPKG](https://unpkg.com/) so that you can load it via a script tag and use the above functions from the `esriLoader` global.
+Typically you would [install the esri-loader package](#install) and then use a module loader/bundler to `import` the functions you need as part of your application's build. However, ES5 builds of esri-loader are also distributed on [UNPKG](https://unpkg.com/) both as ES modules and as a [UMD](http://jargon.js.org/_glossary/UMD.md) bundle that exposes the `esriLoader` global.
+
+This is an _excellent_ way to prototype how you will use the ArcGIS API for JavaScript, or to isolate any problems that you are having with the API. Before we can help you with any issue related to the behavior of a map, scene, or widgets, we will **require** you to reproduce it _outside_ your application. A great place to start is one of the codepens linked below.
+
+#### Using a module script tag
+
+You can load the esri-loader [ES modules directly in modern browsers](https://caniuse.com/#feat=es6-module) using `<script type="module">`. The advantage of this approach is that [any browser that supports `type="module"` also supports ES2015](https://philipwalton.com/articles/deploying-es2015-code-in-production-today/) and most support later features like [`async`/`await`](https://caniuse.com/#search=await). This means you can use all of those features in your script.
```

Comment:
Practically speaking, it's very difficult to find and run a browser version these days that supports `type=module` but NOT `async/await`. But I suppose it's not 100% 1:1, so I see why you used "most" here.

Reply:
exactly, I flip/flopped on that for a bit, but think I came up w/ something better in 1a6ca7013b8a96d717147a51326dc21ac01b8067